subcommands/keys/tuf_updates_review.go

```diff
@@ -101,20 +103,50 @@ func doTufUpdatesReview(cmd *cobra.Command, args []string) {
 				}
 			}
 		}
+	} else if updates.Status == client.TufRootUpdatesStatusNone {
+		fmt.Println("There are no TUF root updates in progress.")
+		if len(updates.Advices.Mandatory) > 0 {
+			fmt.Println(`\nFoundries.io identified some problems with your existing TUF metadata.
+Please, consider these updates in order to keep your TUF metadata functional and secure:`)
+			for _, advice := range updates.Advices.Mandatory {
+				fmt.Printf(" - %s\n", advice.Message)
+			}
+		}
+		if len(updates.Advices.Recommended) > 0 {
+			fmt.Println("\nFoundries.io recommends these updates to improve your existing TUF metadata:")
```

Comment:
I'd drop the branding here also and just say something like "The following updates are recommended..."

Reply:
Ok

Reply:
Done in force-push